doc/dev/design/FABRID.rst

```diff
@@ -0,0 +1,605 @@
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The *IndexIdentifierMap* and *SupportedIndicesMap* are included in a (unsigned) detachable extension in the PCBs for an AS.
+Hashes of these maps are maintained in a Signed AS Entry, such that the authenticity of these maps can be verified.
+The detachable extension can also be present in the PCB, i.e. it does not have to be detached in all cases, e.g. if there are only very few policies.
+If the maps are detached, they can be fetched from the control service of that AS and the received maps can be verified with the hashes.
+To ensure a consistent hash calculation, the key entries of these maps have to be sorted, such that they are accessed in a consistent order.
+
+.. _endhost_policy_selection:
+
+Exposing policies to the endhosts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The path combinator finds the most recent FABRID map per AS among the received segments and subsequently uses this map to find the FABRID
+policies that are available for each interface pair of hops.
+This results in a set of *PolicyIdentifiers* per hop, which can then be used by the application, such as by defining an application parameter
+that then selects the policies to use on the path and hands these to the snet implementation, e.g. ``--fabridpolicy``. 
+
+
+``fabridpolicy`` parameter
+''''''''''''''''''''''''''''
+A custom language is used to make a selection out of the available paths and policies. The basic components of the language are as follows:
+
+* **Identifiers**
+
+  An identifier matches with a specific hop in the path and applies a policy to that hop. Parts of this hop identifier may be a wildcard,
+  such that the identifier can match with multiple hops in the path.
+  An identifier is structured as follows: 
+  ``ISD-AS#IGIF,EGIF@POLICY``,
+  where 
+
+  * ISD can be either the ISD number (e.g. ``1``), or a wildcard (``0``).
+  * AS can be either the AS number seperated by underscores (e.g. ``ff00_0_110``) or a wildcard (``0``).
+  * IGIF can be either the ingress interface number (e.g. ``42``), or a wildcard (``0``).
+  * EGIF can be either the egress interface number (e.g. ``41``), or a wildcard (``0``).
+  * POLICY can be either the policy to apply, where a local policy is denoted as ``L`` + the policy identifier (e.g. ``L100``) and a global policy
+    is denoted by ``G`` + the policy identifier (e.g. ``G100``), a wildcard (``0``), or a rejection ``REJECT``.
+    Rejection means that this path should not be chosen. 
+
+  When used in a query, the identifier evaluates to true when at least a single hop in the path matches the identifier.
+
+* **Concatenations**
+
+  Multiple identifiers can be combined by using a concatenation (and/or parentheses). Concatenations are created by the ``+`` symbol. 
+
+  Example:
+
+  ``(1-0#0,0@G300 + 1-0#0,0@G200)`` applies both policy G300 and policy G200.
+
+  When used in a query, a concatenation evaluates to true when *all* identifiers in the concatenation also evaluate to true.
```

Comment:
Move to section about queries?
It is not clear at that point what a query actually is.